misk-service/src/main/kotlin/misk/ServiceManagerModule.kt

```diff
@@ -76,6 +76,9 @@ class ServiceManagerModule : KAbstractModule() {
     }
 
     val serviceManager = builder.build()
+    if (serviceManagerConfig.debug_service_graph) {
```

Comment:
Would this fail if the no-config constructor is used? Since the val wouldn't exist or be initialized?

Reply:
It shouldn't, the config data class has a default value of `false` for this property, and the no-arg constructor for this module uses a default construction of the config class.